.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,15 @@
+{
+    "configurations": [
+        {
+            "name": "Linux",
+            "includePath": [
+                "${workspaceFolder}/**"
+            ],
```

Comment:
Hmm, I don't think this is a good includePath.

Perhaps we should use compileCommands instead?
```json
            "compileCommands": "${workspaceFolder}/build/compile_commands.json"
```